plugins/catalog/api-report.md

```diff
@@ -416,4 +417,8 @@ export type SystemDiagramCardClassKey =
   | 'componentNode'
   | 'apiNode'
   | 'resourceNode';
+
```

Comment:
Can we fix up these warnings too?

Reply:
Happy to do it, how could I have detected that myself? I didn't see any warning. Can I run a check locally?